src/components/_video/VideoCategoryCard/FeaturedVideoCategoryCard.tsx

```diff
@@ -39,7 +39,8 @@ export const FeaturedVideoCategoryCard: React.FC<FeaturedVideoCategoryCardProps>
 }) => {
   const [hoverRef, isVideoHovering] = useHover<HTMLAnchorElement>()
   const isLoading = !title && !videoUrl && !icon
-  const isPlaying = variant === 'default' ? isVideoHovering : true
+  const isTouchDevice = !!('ontouchstart' in window || navigator.maxTouchPoints)
```

Comment:
Could we use a media query instead? This could work similar way that our `useMediaMatch` does. This should do the trick: `@media (any-pointer: coarse)`